cmake/Config.cmake

```diff
@@ -110,13 +110,25 @@ elseif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 
     execute_process(COMMAND "${CMAKE_CXX_COMPILER}" "--version" OUTPUT_VARIABLE CLANG_VERSION_OUTPUT)
     string(REGEX REPLACE ".*clang version ([0-9]+\\.[0-9]+).*" "\\1" SFML_CLANG_VERSION "${CLANG_VERSION_OUTPUT}")
+
+    execute_process(COMMAND "${CMAKE_CXX_COMPILER}" "-v" OUTPUT_VARIABLE CLANG_COMPILER_VERSION ERROR_VARIABLE CLANG_COMPILER_VERSION)
+
+    if("${CLANG_COMPILER_VERSION}" MATCHES "ucrt")
+        set(SFML_RUNTIME_UCRT 1)
+    endif()
```

Comment:
I've only heard of UCRT MinGW. Is there also a URCT version of Clang someone may use?

Reply:
MinGW is more like the toolchain, within which you can have GCC or Clang as compiler and they can link the universal C runtime (UCRT) or Microsoft Visual Studio C runtime (MSVCRT)